spec/mongo/collection_spec.rb

```diff
@@ -3487,5 +3487,213 @@ def generate
         expect(result).to be_nil
       end
     end
+
+    context 'when arrayFilters is provided' do
```

Comment:
Would you mind moving these tests to be under their respective methods?

So for #update_one, you'd put the block under the context for the method here:
https://github.com/saghm/mongo-ruby-driver/blob/304438edc9680f214b8d67ffae2951c50b602035/spec/mongo/collection_spec.rb#L2441

for #update_many, you'd put it in this context:
https://github.com/saghm/mongo-ruby-driver/blob/304438edc9680f214b8d67ffae2951c50b602035/spec/mongo/collection_spec.rb#L2169

and #find_one_and_update:
https://github.com/saghm/mongo-ruby-driver/blob/304438edc9680f214b8d67ffae2951c50b602035/spec/mongo/collection_spec.rb#L2900